packages/flutter/lib/src/foundation/constants.dart

```diff
@@ -38,3 +38,6 @@ const bool kDebugMode = !kReleaseMode && !kProfileMode;
 /// precision loss in calculations. Differences below this threshold are safe to
 /// disregard.
 const double precisionErrorTolerance = 1e-10;
+
+/// A constant that is true if the application was compiled to JavaScript.
+const bool kIsWeb = identical(1, 1.0);
```

Comment:
Is this where we want to keep this constant? Removing this later will be a breaking change, no?

Reply:
I could leave out the constant for now